python/setup.sh.in

```diff
@@ -0,0 +1,24 @@
+# This file is part of the Acts project.
```

Comment:
Should these be updated to actsvg?

Reply:
No that's fine, actsvg is a subproject of Acts, so we can leave it, I suppose.